src/staking-v3/components/Edit.vue

```diff
@@ -0,0 +1,52 @@
+<template>
+  <div class="wrapper--edit">
+    <div class="row--title">
+      <span> {{ $t('stakingV3.edit') }} </span>
+    </div>
+    <div class="box--links">
+      <!-- Todo: add links -->
```

Comment:
Should we add the logic for owner page in this PR? If so, let's add the links

Reply:
Let's leave this for the future PR